packages/client/v2.graphql

```diff
@@ -0,0 +1,216 @@
+type Query {
+  challenge(request: ChallengeRequest!): AuthChallengeResult!
```

Comment:
I added this subset of the schema to have easy reference to all queries and mutations and to spot if anything is missing, I will remove it later